packages/common/src/services/integrations/cleeng/CleengService.ts

```diff
@@ -130,12 +131,12 @@ export default class CleengService {
       };
     } catch (error: unknown) {
       if (error instanceof Error) {
-        logDev('Failed to refresh accessToken', error);
+        getLogger().debug('CleengService', 'Failed to refresh accessToken', { error });
 
         // only logout when the token is expired or invalid, this prevents logging out users when the request failed due to a
         // network error or for aborted requests
         if (error.message.includes('Refresh token is expired or does not exist') || error.message.includes('Missing or invalid parameter')) {
-          if (!this.logoutCallback) logDev('logoutCallback is not set');
+          if (!this.logoutCallback) getLogger().debug('CleengService', 'logoutCallback is not set');
```

Comment:
I generally recommend to always use braces. Maybe we should update linting to enforce.
```suggestion
          if (!this.logoutCallback) {
            getLogger().debug('CleengService', 'logoutCallback is not set');
          }
```

Reply:
I can update this, but there are probably more of these in the codebase. Shall we do this in a separate update linting PR?